workspaces/liferay-customer-workspace/client-extensions/liferay-customer-site-initializer-code/site-initializer/fragments/group/liferay-customer/customer-portal/fragments/callback-form-sla-text/index.html

```diff
@@ -1,6 +1,7 @@
 [#assign
 	displayObject = (request.getAttribute("LAYOUT_DISPLAY_PAGE_OBJECT_PROVIDER").getDisplayObject())!{}
 	objectEntryJSON = jsonFactoryUtil.createJSONObject()
+	languageId = themeDisplay.getLanguageId()
```

Comment:
@JoMaAlves I dont think this line is needed anymore, but the rest of the changes look great!

Reply:
Oops, my bad

Reply:
Done!